app/src/main/java/com/steamclock/steamclogsample/MainActivity.kt

```diff
@@ -205,7 +207,16 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun testLogDump() = GlobalScope.launch(Dispatchers.Main) {
-        binding.demoText.text = SteamcLog.getFullLogContents()
+        var logMetaData = "Log files (Last modified descending): \n\n"
+        SteamcLog.getAllLogFiles(LogSort.LastModifiedDesc)?.forEachIndexed { index, file ->
+            val lastModified = SimpleDateFormat("dd-MM HH:mm:ss").format(Date(file.lastModified()))
+            val fileSizeInBytes = file.length() / 1024
+            val fileData = "$index: $lastModified ${file.name} (${fileSizeInBytes}b)"
+            logMetaData += fileData + "\n"
+        }
+
+        logMetaData += "\nFull Content:\n\n" + SteamcLog.getFullLogContents()
+        binding.demoText.text = logMetaData
```

Comment:
Updating sample app to print out the list of log files so I can properly test the auto rotation feature